client/gefyra/__main__.py

```diff
@@ -250,30 +256,48 @@ def telemetry_command(on, off):
         logger.info("Invalid flags. Please use either --off or --on.")
 
 
+def prepare_cargo_endpoint(host: str, port: str):
+    if host and not port:
+        return f"{host}:31820"
+    if port and not host:
```

Comment:
First, I couldn't find a reason to justify changing the signature of `ClientConfiguration`. Yet, we do have some code to automatically set the host. So it would be Ok to just set the port like:
`gefyra run --port 31821`
and it would try to connect to `172.0.60.1:31821`, for example.

I'd say let's change the signature of `ClientConfiguration`, replace the argument `cargo_endpoint: str` with
`cargo_endpoint_port: str = 31820` and `cargo_endpoint_host: str = None` and we could keep that code out of the CLI.